executor/executor.go

```diff
@@ -166,13 +173,14 @@ func newBaseExecutor(ctx sessionctx.Context, schema *expression.Schema, id fmt.S
 // return a batch of rows, other than a single row in Volcano.
 // NOTE: Executors must call "chk.Reset()" before appending their results to it.
 type Executor interface {
+	This() *baseExecutor
 	Open(context.Context) error
 	Next(ctx context.Context, req *chunk.RecordBatch) error
 	Close() error
 	Schema() *expression.Schema
 
-	retTypes() []*types.FieldType
-	newFirstChunk() *chunk.Chunk
+	// retTypes() []*types.FieldType
```

Comment:
How about removing them from the code instead of commenting them out?

Reply:
Done